### PR TITLE
[MySQL] Migrate MySQL module to MySQL flexible server GA API

### PR DESCRIPTION
**Pull request by DaeunYim**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by DaeunYim** [comment]
@isra-fel We are GAing MySQL flexible server module. We have updated GA Swagger API and added some more features. Now PowerShell needs to be updated to based on that. We also want to include these to CloudShell once the product officially hit GA
The updated API is here
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2021-05-01-preview
(It's preview and we are making stable version of the same API soon)

File: src/MySql/readme.md [hunk 1]
```diff
@@ -115,6 +112,10 @@ directive:
       verb: New$|Update$
       variant: ^(?!.*?Expanded)
     hide: true
+  - where:
+      verb: Get$
+      subject: ^Configuration$|
```
Comment (isra-fel) [inline src/MySql/readme.md:117]: I think the tailing `|` was by mistake
```suggestion
      subject: ^Configuration$
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @DaeunYim , a reminder to re-record test cases, and update help documents

File: src/MySql/readme.md [hunk 2]
```diff
@@ -115,6 +112,10 @@ directive:
...
+    remove: hide
```
Comment (isra-fel) [inline src/MySql/readme.md:118]:
```suggestion
    remove: true
```

File: src/MySql/readme.md (hunk 1 above)
Comment (DaeunYim) [reply src/MySql/readme.md:117]: @isra-fel, Hi, I'm little running behind bc of workload but will make sure to finish these by EOW.. 
Thanks a lot for prompt reviews

**Comment by DaeunYim** [comment]
Couldn't finish this PR by code complete day. I'm pushing this to next milestone and submitted a fix PR for exisitng incidents (Only changing default values and updated bug in a cmdlet doc)
https://github.com/Azure/azure-powershell/pull/15536

**Pull request closed without merging**